# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: architecture/avr32/package/.../inet.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Required by libxcb.

  - Rene Rebe <rene@exactcode.de>

diff -urN uclibc/include/netdb.h uclibc-patched/include/netdb.h
--- uclibc/include/netdb.h      Tue Feb 26 03:32:21 2002
+++ uclibc-patched/include/netdb.h      Tue Mar 19 08:58:54 2002
@@ -442,4 +442,7 @@
 # define AI_CANONNAME	0x0002	/* Request for canonical name.  */
 # define AI_NUMERICHOST	0x0004	/* Don't use name resolution.  */
+# define AI_V4MAPPED	0x0008	/* IPv4-mapped addresses are acceptable.  */
+# define AI_ALL	0x0010	/* Return both IPv4 and IPv6 addresses.  */
+# define AI_ADDRCONFIG 0x0020	/* Use configuration of this host */
 
 /* Error values for `getaddrinfo' function.  */
